frontend/app/src/App.test.tsx

```diff
@@ -2797,6 +2802,9 @@ describe("App", () => {
         widgetStateManager,
         "sendUpdateWidgetsMessage"
       )
+      // Somehow the spy still registers one previous call to the function.
+      // To work around this, we clear the spy
+      sendUpdateWidgetsMessageSpy.mockClear()
```

Comment:
The vitest update caused an error in this test. Somehow, the spy was registering the previous call to `sendUpdateWidgetsMessage`  even though it was not yet spying on it. Therefore, we explicitly clear it before simulating the new connection.